phpcs.xml.dist

```diff
@@ -3,7 +3,6 @@
     <rule ref="PSR12"/>
 
     <file>src</file>
-    <file>tests</file>
```

Comment:
I don't necessarily agree with this, but it's fine for now, once I decide to bring this back to the DMS Coding Standard I can address this